libbeat/asset/asset.go

```diff
@@ -44,9 +44,10 @@ func Asset() string {
 `))
 
 type Data struct {
-	License string
-	Beat    string
-	Name    string
-	Data    string
-	Package string
+	License  string
+	Beat     string
+	Name     string
+	Priority int
```

Comment:
I think it would be nice to have some named constants that can be used in the generated fields.go. Something like

```
type Priority int32

const (
    Highest Priority = 0
    LibbeatFieldsPri Priority = 1
    BeatFieldsPri Priority = 2
    ModuleFieldsPri Priority= 10
    Lowest Priority = math.MaxInt32
```

Then when viewing the generate fields.go I will see `asset.ModulePri` or whatever you name them.